### PR TITLE
Chat 149/js desktop ex

### PR DESCRIPTION
**Pull request by crgbaumgart**
- Added demo steps/examples for: user-presence, message history, and markdown plugin.
- Tore JS out of HTML files.

### CODE REVIEW

File: javascript/chat.html [hunk 1]
```diff
@@ -30,68 +30,17 @@
                         <input type="text" class="form-control message" placeholder="Your Message..." id="chat-input" onkeypress="checkSubmit(event)">
                         <span class="input-group-btn"><button class="btn btn-primary" type="submit" onclick="sendChat()">Send</button></span>
                     </div>
+                    <!-- <div class="row message">
```
Comment (madisonsmith) [inline javascript/chat.html]: If this is just commented out can it be removed?

File: javascript/js_example.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+# PubNub ChatEngine - Javascript Desktop Ex.
```
Comment (madisonsmith) [inline javascript/js_example.md:1]: So good. Let's make sure we get this to Joe!

File: javascript/chat.html (hunk 1 above)
Comment (crgbaumgart) [reply javascript/chat.html]: All gone!

**Review by madisonsmith — APPROVED** [review]
🎈

**Comment by crgbaumgart** [comment]
Codacy has lost its mind due to not knowing that this example is meant for browsers. Ignoring.

**Pull request merged**